### PR TITLE
WL-4031 Added spinning wheel when Remove Participant group is clicked.

### PR DESCRIPTION
**Pull request by ouit0408**
Added spinny wheel image for remove participant  button in 'chef_site_siteInfo-list.vm'

### CODE REVIEW

File: site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-list.vm [hunk 1]
```diff
@@ -523,10 +527,13 @@ function printPreview(target){
 						<td>$validator.escapeHtml($group.name)</td>
 						<td>
 							#if ($allowUpdateSiteMembership)
-							<form name="removeGroupForm" class="inlineForm" method="post" action="#toolForm("SiteAction")">
+							<form id="removeGroupForm" name="removeGroupForm" class="inlineForm" method="post" action="#toolForm("SiteAction")">
 								<input type="hidden" name="groupId" value="$group.id"/>
 								<input type="hidden" name="sakai_csrf_token" value="$sakai_csrf_token" />
 								<input type="submit" name="eventSubmit_doRemoveGroup" value="$tlang.getString("sitegen.siteinfolist.remove")"/>
+								<span id="spinnerRemove" style="display:none;">
+									<img class="loader" src="/../../library/newspicker/css/images/ajax-loader.gif" alt="Loading"/>
```
Comment (buckett) [inline site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-list.vm]: Much better to just have `/library/...` as it the URL of the page includes more parts the link will break.

File: site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-list.vm [hunk 2]
```diff
@@ -523,10 +527,13 @@ function printPreview(target){
 						<td>$validator.escapeHtml($group.name)</td>
 						<td>
 							#if ($allowUpdateSiteMembership)
-							<form name="removeGroupForm" class="inlineForm" method="post" action="#toolForm("SiteAction")">
+							<form id="removeGroupForm" name="removeGroupForm" class="inlineForm" method="post" action="#toolForm("SiteAction")">
```
Comment (buckett) [inline site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-list.vm]: You shouldn't have multiple elements in the same page with the same ID (as this is in a loop).

**Comment by buckett** [comment]
Still waiting for response to comments.

**Comment by buckett** [comment]
Still has comments outstanding.

**Comment by ouit0408** [comment]
sorry I missed the previous comments , will have a look soon.

**Comment by ouit0408** [comment]
Implemented the review comments and have pushed the code.

**Pull request merged**